spec/interactor/organizer_spec.rb

```diff
@@ -41,13 +41,15 @@ module Interactor
       before do
         allow(instance).to receive(:context) { context }
         allow(organizer).to receive(:organized) {
-          [interactor2, interactor3, interactor4]
+          [interactor2, :mapping1, interactor3, :mapping2, interactor4]
```

Comment:
From my point of view, suggested solution solves not only mappings issue; it's more generic. For example, it supports a use case I've faced with: outputting different strings to stdout after successful pass of an interactor within a chain. The strings are different from one organizer to another, so it doesn't make sense to output the in the interactor. It could be perfectly solved with this PR.
Thus I suggest providing more generic naming here. Maybe just `method_name1` instead of `mapping1` could fit?